src/test/jtx/impl/AMM.cpp

```diff
@@ -300,7 +300,7 @@ AMM::expectTradingFee(std::uint16_t fee) const
 {
     if (auto const amm =
             env_.current()->read(keylet::amm(asset1_.issue(), asset2_.issue()));
-        amm && amm->getFieldU16(sfTradingFee) == fee)
+        amm && (*amm)[~sfTradingFee].value_or(0) == fee)
```

Comment:
This can safely be changed back to the original code `amm && amm->getFieldU16(sfTradingFee) == fee)` or `amm && (*amm)[sfTradingFee] == fee)`. If the field isn't set, it'll return 0.